.gitignore

```diff
@@ -2,4 +2,10 @@ bentoo
 BenchmarkJava*
 reality-check*
 JulietCSharp*
-tool_runners
+sast-rules/
+tool_runners/
+
+.idea/
```

Comment:
It is better to add files and directories that are not related to the repository to `.git/info/exclude`.